### DISCUSSION TITLE
Display 100s or 50s if Sliderends/Reverse Arrows/etc are missed

### DISCUSSION THREAD

**Discussion by thacer13** [ppy/osu#15977] [Ideas] [answered]
Visual feedback is lacking (compared to stable) when sliders aren't hit properly. As it is, tapping the slider head correctly but fully missing the slider body does not visually produce a hit100 or hit50 at its end, only a hit300 for (and on top of) its head. I've noticed that behavior confusing newer players firsthand, as HP starts to drain and accuracy starts to drop "mysteriously".

~~Edit: did not check for this in any platform other than Linux~~

Edit 2: changed categories since this isn't a bug.

**Comment by integerrr** [comment]
Are you sure? Off the top of my head that's not the case, do you have a video to support this?

**Reply by plaguemask** [reply]
Not OP but here's a video showing it (version 2021.1207.0)

https://user-images.githubusercontent.com/66150801/145078353-bbbad779-c8b7-4619-bec8-8d96b9f209d9.mp4

**Reply by thacer13** [reply]
Thanks, it happens in any map just like that.

**Comment by bdach** [comment, marked as answer]
so a couple of things here:

* this is only the case with the classic mod off. with the classic mod on (in particular with slider head accuracy off), the entire slider is judged proportionally to % of nested objects hit. so with classic mod, there is no need to show anything on slider ends and repeats.
* the "classic mod off" case definitely needs work. a repeat is a "large tick" in the new scoring vernacular, and a tail is a "small tick". the visual display could be better but 100s/50s should not be displayed, because that would be misleading; ticks can only be hit or missed, and give a fraction of the points that normal objects do. see more [here](https://github.com/ppy/osu/wiki/Scoring)

**Reply by integerrr** [reply]
Are you trying to say ticks can only be either hit or missed as they have no accuracy component?

Edit: sorry I thought the sentence got cut off as it didn't display the last bit about ticks

**Reply by bdach** [reply]
they contribute to "accuracy" in the sense of the percentage at the top of the screen (as that percentage is a ratio of points hit to points available), but themselves they are binary in their judgement, yes. you can't "sort of hit" a tick, it's either a hit or a miss.

**Reply by thacer13** [reply]
If it has any relevancy, may I ask how it is done/shown in stable's ScoreV2?

Furthermore, even if 100s & 50s aren't the most logical way of representing missed "ticks" as you say, saying it is misleading is an overstatement from a player's perspective, unlike the current lack of any visual feedback which certainly is. Not to mention, it's now called "ok" and "meh" anyway.

**Reply by bdach** [reply]
i'm pretty sure stable numerically judges the slider as a whole, full stop, regardless of scorev2.

and note that i did not at any point claim that the tick hit/miss shouldn't be shown at all, it just shouldn't be shown as a 100/50. it can be shown in any other way. in fact *standard* slider ticks already flash red on miss - a similar concept could be deployed in the case of repeats/tails.

**Reply by integerrr** [reply]
In sv1 and sv2, sliders are judged at their ends, so the 300/100/50 judgement would make sense as it represents overall how accurately the slider as a whole was hit, this is not how lazer slider works. If slider ticks/repeats shows 100/50 it simply doesn't make sense, because it would imply you somehow "kind of hit" the ticks

**Reply by thacer13** [reply]
Alright, I just fear that adding another element will either get bloated or be too discreet (as the reddening ticks are imo), and still don't see why the way the slider is composed and calculated should make this issue that's purely visual(?) so convoluted.

I didn't intend to make this thread be about a feature request, but if that's really how it has to be I hope something ingeniously simple is figured out soon, trust me when I say the game is fundamentally confusing at this point in time, particularly for new users.

**Reply by integerrr** [reply]
Precisely because this isn't a "purely visual" issue. For now, if you need the dropped slider ends/ticks to give 50/100s use classic mod

**Reply by peppy** [reply]
Just to confirm, if we are to show something, it will likely be a cross on missing these elements (and something similar to 10/30 score points appearing in stable when you hit them successfully). It will not be included in the judgement because it isn't. That's part of the simplification / fixing process we've applied to the new scoring system.
